### PR TITLE
Create a new view to show all the events attached

### LINKED ISSUE #42: Fixing issues
# REQUIREMENTS
Solve issues with player routes and events

### PR DESCRIPTION
**Pull request by MauricioNC**
# Description
Create a new view to show all the events attached

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

File: EasyLearn/app/controllers/search_controller.rb [hunk 1]
```diff
@@ -3,5 +3,9 @@ class SearchController < ApplicationController
   def index
     @event = Event.where(['title LIKE ?', "%#{(params[:query])}%"])
     @video = Video.where(['title LIKE ?', "%#{(params[:query])}%"])
+
```
Comment (VictorElizalde) [inline EasyLearn/app/controllers/search_controller.rb]: Remove extra line

**Review by VictorElizalde — APPROVED** [review]
Just erase the extra line but I already approved it.

File: EasyLearn/app/views/search/index.html.erb [hunk 2]
```diff
@@ -49,4 +36,17 @@ function hide(clas) {
...
+  var x = document.getElementsByClassName(clas);
+  for (let i=0;i<x.length;i++){
+    if (x[i].style.display === "none") {
+    x[i].style.display = "flex";
+    } else {
+    x[i].style.display = "none";
+    }
+  }
+}
+</script>
```
Comment (YilKanda) [inline EasyLearn/app/views/search/index.html.erb]: What do you think about put this code in an external file?

**Review by YilKanda — APPROVED** [review]

File: EasyLearn/app/controllers/search_controller.rb (hunk 1 above)
Comment (MauricioNC) [reply EasyLearn/app/controllers/search_controller.rb]: Done. Thanks

**Review by JuanequeX — APPROVED** [review]
LGTM

File: EasyLearn/app/views/search/index.html.erb (hunk 2 above)
Comment (MauricioNC) [reply EasyLearn/app/views/search/index.html.erb]: Done.

**Pull request merged**